collections/_article/introducing-the-improved-ufbx-importer-in-godot-4-3.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Introducing the improved ufbx importer in Godot 4.3"
+excerpt: "Godot 4.3 now includes native .fbx support"
+categories: ["progress-report"]
+author: Lyuma
+image: /storage/blog/covers/PlaceHolder.webp
+date: 2099-09-09 00:00:00
+---
+
+Thanks to [a very welcome contribution by Samuli Raivio (bqqbarbhg) and Ernest Lee (fire)](https://github.com/godotengine/godot/pull/81746), Godot 4.3 now includes native .fbx support. This was made possible using Samuli’s free and open source [ufbx single-file .fbx library](https://github.com/ufbx/ufbx). The ufbx importer will be used by default for all newly imported .fbx files in Godot 4.3.
+
+# What’s new in ufbx?
+
+First and foremost, it will no longer be necessary to download the external FBX2glTF converter in order to use .fbx files in Godot 4.3 projects. This also enables .fbx support on all platforms Godot supports, as well as script access using the new FBXDocument API.
+
+The new importer also contains many technical improvements and changes:
+* Tested against content created in [a wide range](https://github.com/ufbx/ufbx/tree/master/data) of professional 3D DCC (digital content creation) tools
+* Imported scenes no longer have an extraneous "RootNode" top-level node in the scene.
+* Unit conversion (such as centimeters to meters)  is now applied to the mesh itself rather than assigned to the node scale of top-level nodes.
```

Comment:
```suggestion
* Unit conversion (such as centimeters to meters) is now applied to the mesh itself rather than assigned to the node scale of top-level nodes.
```